hack/install.yaml

```diff
@@ -68,30 +68,45 @@ spec:
           type: object
         spec:
           properties:
+            assemblyPhase:
+              type: string
             componentKinds:
               items:
-                properties:
-                  group:
-                    type: string
-                  kind:
-                    type: string
                 type: object
               type: array
             description:
               type: string
-            assemblyPhase:
-              type: string
-              enum:
-              - Pending
-              - Succeeded
-              - Failed
             info:
               items:
                 properties:
                   name:
                     type: string
                   value:
                     type: string
+                  valueFrom:
+                    properties:
+                      ingressRef:
+                        properties:
+                          host:
+                            type: string
+                          path:
+                            type: string
+                        type: object
+                      secretRef:
```

Comment:
I think this should be `secretKeyRef` to match what's used in the podspec. Also there should be a `configMapKeyRef` too, right?

Reply:
Good catch, thanks! I missed updating the json annotation.